packages/oasis-actions/src/helpers/ajna/index.ts

```diff
@@ -103,3 +103,10 @@ export const getAjnaEarnActionOutput = async ({
 
 export const resolveAjnaEthAction = (isUsingEth: boolean, amount: BigNumber) =>
   isUsingEth ? ethers.utils.parseEther(amount.toString()).toString() : '0'
+
+export const calculateAjnaApyPerDays = (amount: BigNumber, apy: BigNumber, days: number) =>
+  // converted to numbers because BigNumber doesn't handle power with decimals
+  new BigNumber(
+    (amount.toNumber() * Math.E ** (apy.toNumber() * (days / 365)) - amount.toNumber()) /
+      amount.toNumber(),
```

Comment:
be careful not to overflow the number, maybe try doing the operation on BigNumbers